src/components/view/news/news.ts

```diff
@@ -0,0 +1,59 @@
+import { Article } from '../../../types';
+import './news.css';
+
+class News {
+    draw(data: Article[]) {
+        const news = data.length >= 10 ? data.filter((_item, idx) => idx < 10) : data;
+
+        const fragment = document.createDocumentFragment();
+        const $newsItemTemp = document.querySelector('#newsItemTemp') as HTMLTemplateElement;
+
+        news.forEach((item, idx) => {
```

Comment:
wow, this cycle is huge. Please, refactor it. Split the logic in separate functions